frontends/web/src/components/balance/balance.tsx

```diff
@@ -15,22 +15,20 @@
  * limitations under the License.
  */
 
-import { PropsWithChildren } from 'react';
+import { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
 import { IBalance } from '../../api/account';
 import { FiatConversion } from '../../components/rates/rates';
-import { translate, TranslateProps } from '../../decorators/translate';
 import style from './balance.module.css';
 
-interface BalanceProps {
+interface Props {
     balance?: IBalance;
 }
 
-type Props = BalanceProps & TranslateProps;
-
-function Balance({
-    t,
+export const Balance: FunctionComponent<Props> = ({
```

Comment:
@benma do you ack to renaming 'CompomentNameProps' to just 'Props' or is it helpful if it stays, i.e. in this case `BalanceProps`?

Reply:
sure, ACK.